app/models/group.rb

```diff
@@ -19,7 +19,21 @@ class Group < ActiveRecord::Base
   has_many :leaders, -> { where(group_members: { leader: true }) },
            through: :group_members, source: :user
 
+  after_destroy :destroy_notifications
+
   def led_by?(user)
     leaders.include? user
   end
+
+  def notifications
+    Notification.where('uniqueid ilike ?', '%new_group%').select do |n|
```

Comment:
It seems like there should be a way to directly query for this instead of doing one query and then selecting, but I couldn't figure it out.